docs/administration/robot-accounts/_index.md

```diff
@@ -5,39 +5,27 @@ weight: 40
-
-* List Repository
-* Pull Repository
-* Push Repository
-* Delete Repository
-* Read Artifact
-* List Artifact
-* Delete Artifact
-* Create Artifact Label
-* Delete Artifact Label
-* Create Tag
-* Delete Tag
-* List Tag
-* Create Scan
-* Stop Scan
-
-Depending on your needs you can assign a combination of these permissions to a system robot account to perform your desired tasks through the OCI client or Harbor API. Robot Accounts cannot log in to the Harbor interface.
-
-You are also able create project scope robot account that only have access to a single project. Read more about [project robot accounts](../../working-with-projects/project-configuration/create-robot-accounts/).
+For each system robot account you are able to assign the system permissions and specify the projects it covers. And for each of the specified projects you are able to assign the project permissions
+
+You can refer to the [**Permission References**](#permission-references) to assign a combination of these permissions to a system robot account to perform your desired tasks through the OCI client or Harbor API. Robot Accounts cannot log in to the Harbor interface.
+
+You are also able to create project scope robot account that only have access to a single project. Read more about [project robot accounts](../../working-with-projects/project-configuration/create-robot-accounts/).
 
 ## View System Robot Accounts
 
 1. Log into your harbor instance as an administrator.
 1. Go to **Robot Accounts** item under **Administration**.
 
-![System robot account page](../../img/system-robot-account-page.png)
+![System robot account page](../../img/robotaccount/system-robot-account-page.png)
 
 This page lists all available system robot accounts for your Harbor instance. The table lists the following information for each system robot account,
 
 * The name of a system account. This is derived from robot account prefix configured for your Harbor instance and the name assigned to the account when it was created. A robot account name follows the format `<prefix><account_name>`. If you use the search function on this page, you only need to search for the account name without the prefix.
 * Enabled status shows if an account is enabled or deactivated.
-* The number of projects an account is associated with. To see a full list of the projects an account is associated with, click on the **Project(s)** link.
+* The number of system permissions an account is assigned to. To see a full set of the assigned system permissions, click on the **PERMISSIONS** link.
+
+    ![View all the system permissions](../../img/robotaccount/view-system-permissions.png)
+* The number of projects an account is associated with. To see a full list of the projects an account is associated with, click on the **PROJECT(S)** link.
 
     ![View list of all projects associated with a system robot account](../../img/list-robot-account-projects.png)
 
```

Comment:
* The created time shows when the robot account was created.
* The account expiration time. Calculated based on the created time and the expiration time set when creating the robot account.
* The manually added description for the system robot account.